README.md

```diff
@@ -1 +1,275 @@
-# capstone-onlie-ticket-
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <!-- <img src="" alt="logo" width="140"  height="auto" /> -->
+  <br />
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Online Ticket] <a name="about-project"></a>
+> This is my first capstone project. I design a web page for purchasing tickets online to attend programes, shows, concert parties, etc.
+> The project was based on Cindy Shin's design.
+
```

Comment:
- [ ] There's an important feature missing in your project's documentation: The video presentation. As requested by the previous reviewer, please make sure to add a video presentation showcasing the features of your app as well as some piece of code that makes you proud! 🤓 

According to the requirements:

![image](https://user-images.githubusercontent.com/77069011/218207186-daa1d0d2-e030-468b-81ea-658e487f6959.png)

Please don't forget to add it in your PR description **OR/AND** in your README file. 🤖 🤓

Reply:
@anagudelogu 

Please below is the shot of the video presentation in my README.md, which perhaps you might have missed. Kindly confirm .


![image](https://user-images.githubusercontent.com/65625147/218236756-22ec835d-578b-4228-a0e3-f7b85c65376e.png)

Reply:
You're right @shafiuyushawu, I'm sorry, I missed it 😓. kindly ignore this requirement and submit the project again once you finish the other suggested changes for the next reviewer to give you feedback. Cheers and happy coding. 🍾